lib/Store/CouchDB.pm

```diff
@@ -1,11 +1,13 @@
 package Store::CouchDB;
 
+use strict;
+use warnings;
```

Comment:
Regarding Moose or Mouse, I don't want to change that, since I can't evaluate the impact